ConsoleApp1/LibraryCard.cs

```diff
@@ -0,0 +1,153 @@
+                }
+            }
+
+            if (searchSuccess)
+            {
+                Console.WriteLine("Provide the index of the book to be assigned to your library card. 0 - to leave.");
+                int option = int.Parse(Console.ReadLine());
+                if (option == 0) return;              
+                assignedBooks.Add(searchResult[option-1]);
+                allReadBooks.Add(searchResult[option - 1]);
+                readBooks++;
+            }
+            else
+            {
+                Console.WriteLine("The book you searched is not found. Look through other books available in the library");
+                Library.ListofBooks();
+            }
+        }
+
+        /// <summary>
+        /// Remove a book from a particular card 
+        /// </summary>
+        public void RemoveBook()
+        {
+            Console.WriteLine("Provide the index of the book to be removed from your library card. 0 - to leave.");
+            for (int i = 0; i < assignedBooks.Count; i++)
+            {
+                Console.WriteLine($"#{i+1} {assignedBooks[i].BookInfoShort()}");
+            }
+
+           int query = int.Parse(Console.ReadLine());
+
+            if (query == 0)
+            {
+                return;
+            }
+            else if (query > assignedBooks.Count || query < 0)
+            {
+                Console.WriteLine("Please, select the book from the list.");
+            }
+            else
+            {
+                assignedBooks.RemoveAt(query-1);
+            }
+        }
+
+        /// <summary>
+        /// Display basic info about card owner
+        /// </summary>
+        public void LibraryCardShortInfo()
+        {
+            Console.WriteLine("**********************");         
+            Console.WriteLine($"Reader id: {id}");
+            Console.WriteLine(reader.ReaderInfo());
+            Console.WriteLine($"Number of assigned books: {assignedBooks.Count}");
+            Console.WriteLine("**********************");
+        }
+
+        /// <summary>
+        /// Display the full info about card owner,including the list of assigned books with short description
+        /// </summary>
+        public void LibraryCardFullInfo()
+        {
+            Console.WriteLine("**********************");
+            Console.WriteLine($"Reader id: {id}");
+            Console.WriteLine(reader.ReaderInfo());
+            Console.WriteLine("**********************");
+            Console.WriteLine($"Number of all read books {ReadBooks}");
+            Console.WriteLine($"Number of assigned books: {assignedBooks.Count}");
+            Console.WriteLine("The list of assigned books:");
+
+            foreach (var book in assignedBooks)
+            {
+                Console.WriteLine(book.BookInfoShort()); 
+            }
+        }
+        /// <summary>
+        /// Display the list of all books that card owner read
+        /// </summary>
+        public void AllReadBooks()
```

Comment:
add a verb to method's name, (DisplayAllReadBooks, Get, Show)

Reply:
yes, you are right, corrected